docs/configuration.rst

```diff
@@ -60,244 +64,38 @@ MAX_RESPONSE_LENGTH
-
-.. figure:: images/auth0-ga4gh-api-details.png
-   :alt: authorize\_client
-
-+++++++++++++++++++++++++++++
-Creating Database Connections
-+++++++++++++++++++++++++++++
-
-Go to ``Connections-Database`` tab and click ``Create DB Connection``. I
-disabled sign ups because I want to have control over user creation
-allowing only certain people/software to have access on server.
-
-.. figure:: images/auth0-new-database.png
-   :alt: create\_database
-
-
-In ``Clients`` tab, activate the connection between the Client and the
-Database.
-
-.. figure:: images/auth0-database-details.png
-   :alt: client\_db
-
-++++++++++++++
-Creating Users
-++++++++++++++
-
-Go to ``Users`` tab and click ``Create User``. Fill the fields.
-``Connection`` should be same one previously created. An email will be
-send requesting to verify. I recommend creating your own user for
-testing purposes.
-
-.. figure:: images/auth0-create-user.png
-   :alt: create\_user
-
-+++++++++++++++++++++++++++++++++++++++++++
-Setting up GA4GH server with authentication
-+++++++++++++++++++++++++++++++++++++++++++
-
-Append the following server configuration values to ``config.py`` file
-(`reference <https://github.com/ga4gh/server/pull/1470>`__). Remember to
-update the fields correctly. Create a SECRET\_KEY. I didn’t understand
-the field ``AUTH0_AUTHORIZED_EMAILS``. I just used the same email I used
-to login into Auth0 website.
-
-::
-
-    AUTH0_ENABLED = True
-    SECRET_KEY = "super_secret"
-    AUTH0_SCOPES = "openid email"
-    AUTH0_CALLBACK_URL = "http://192.168.0.5/callback"
-    AUTH0_HOST = "USER.auth0.com"
-    AUTH0_CLIENT_ID = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    AUTH0_CLIENT_SECRET = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    AUTH0_AUTHORIZED_EMAILS = "name@host.com"
-
-Restart the server.
-
-+++++++
-Testing
-+++++++
-
-Finally access server landing page. It will show an error and a link to
-login. At login page, use the same email and password used to create a
-user. Note that it does not have an option to create a user (it can be
-changed).
-
-.. figure:: images/auth0-testing.PNG
-   :alt: login
-
-
-After login you will see the token page.
-
-.. figure:: images/auth0-token.png
-   :alt: token
-
-
-------------------------
-OpenID Connect Providers
-------------------------
-
-The server can be configured to use OpenID Connect (OIDC) for authentication.
-As an example, here is how one configures it to use Google as the provider.
-
-Go to https://console.developers.google.com/project and in create a project.
+------------------
+Docker Deployment
+------------------
 
-.. image:: images/Create_project.png
+Refer to this page for Docker installation instructions
 
-Navigate to the project -> APIs & auth -> Consent Screen and enter a product
-name
+https://github.com/CanDIG/candig_compose
 
-.. image:: images/Consent_screen_-_ga4gh.png
 
-Navigate to project -> APIs & auth -> Credentials, and create a new client ID.
+------------------
```

Comment:
Do we have any info in the document where the acl has to be placed?

Reply:
no not yet i dont think so